### PR TITLE
implement milestone 2

### PR DESCRIPTION
**Pull request by onedebos**
Implemented the Display and ButtonPanel features in this Commit.

### CODE REVIEW

File: src/components/Display.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+class Display extends Component {
+  static propTypes = {
+    result: PropTypes.string
+  };
+  render() {
+    return (
+      <div className="display-component">
+        <div>{this.props.result}</div>
+      </div>
```
Comment (shloch) [inline src/components/Display.jsx]: Use `propTypes` to ensure that `result` is a string
also use `defaultProps` to ensure that default value is '0'

File: src/components/Button.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+class Button extends Component {
+  static propTypes = {
+    buttonName: PropTypes.string
+  };
+  render() {
+    return (
+      <div>
+        <div>{this.props.buttonName}</div>
+      </div>
```
Comment (shloch) [inline src/components/Button.jsx]: use 'propTypes` to ensure `buttonName` is a string

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @onedebos 
nice job so far, please dont forget to to alway add short description on pull request link of what you done so far.
Its great so far what you've done........
## [Changes required]
Few changes to make below and file for last review
Let's do this

**Comment by onedebos** [comment]
Implemented the Display and ButtonPanel features in this commit. Made sure props were strings using PropTypes according to TSE requirements.

File: src/components/Button.jsx [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+  );
+};
 Button.propTypes = {
-  buttonName: PropTypes.string
+  buttonName: PropTypes.string,
+};
+
+Button.defaultProps = {
+  buttonName: '',
 };
```
Comment (shloch) [inline src/components/Button.jsx]: i dont think we need this one, get rid of it

**Review by shloch — APPROVED** [review]
## Hello @onedebos 
Good job, Just delete the one on the comment below before merging
Do that before you merge 
Happy coding

**Pull request merged**